msi/GitChooseEnvironmentDlg.wxs

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
So, would you use the Feature or CustomizeDlg in that dialog set? I looked into this tonight and it doesn't really seem to fit. The user experience isn't what I had hoped for.

The one thing that turned me off to the dialog set was the addition of properties to specify whether the install was per-machine or per-user. For this application it's always per-machine. (Though, let's check my assumptions: does it have to be? Should it be?) _Assuming_ we should always install per-machine, we could get rid of the `InstallScopeDlg`. Since the `CustomizeDlg` doesn't really have a great UX for what we're trying to accomplish (and rolling our own is crazy), I'd strip out that dialog. Then I'm left with the custom dialogs I already created. And finally, we come to right where I am now.

Anyway, I guess this was my thought process for how I arrived at where I am now. But, I'm open to hearing ideas.